### PR TITLE
feat: add active block number

### PR DESCRIPTION
**Pull request by nud3l**

### CODE REVIEW

File: src/parachain/system.ts [hunk 1]
```diff
@@ -8,14 +8,23 @@ export interface SystemAPI {
...
-    async getCurrentBlockNumber(): Promise<BlockNumber> {
+    async getCurrentBlockNumber(): Promise<number> {
+        const head = await this.api.rpc.chain.getFinalizedHead();
+        return await (await this.api.query.system.number.at(head)).toNumber();
+    }
+
+    async getCurrentActiveBlockNumber(): Promise<number> {
         const head = await this.api.rpc.chain.getFinalizedHead();
-        return await this.api.query.system.number.at(head);
+        return await (await this.api.query.security.activeBlockCount.at(head)).toNumber();
```
Comment (savudani8) [inline src/parachain/system.ts:28]: The first `await` is not required, but it doesn't break anything

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@interlay/polkabtc",
-  "version": "0.15.6",
+  "version": "0.16.0",
```
Comment (savudani8) [inline package.json:3]: 0.15.7 will soon be merged, hope it won't cause a conflict

File: src/parachain/system.ts (hunk 1 above)
Comment (nud3l) [reply src/parachain/system.ts:28]: uh, that was the auto-complete, I'll fix that

File: package.json (hunk 2 above)
Comment (nud3l) [reply package.json:3]: yeah, no worries I'll fix that

**Pull request merged**